packages/neon/neon/lib/src/platform/android.dart

```diff
@@ -5,22 +5,44 @@ import 'package:path/path.dart' as p;
 import 'package:path_provider/path_provider.dart';
 import 'package:permission_handler/permission_handler.dart';
 
+@immutable
 @internal
-class AndroidNeonPlatform extends NeonPlatform {
-  AndroidNeonPlatform()
-      : super(
-          canUseWebView: true,
-          canUseQuickActions: true,
-          canUseSystemTray: false,
-          canUseWindowManager: false,
-          canUseCamera: true,
-          canUsePushNotifications: true,
-          getApplicationCachePath: () async => (await getTemporaryDirectory()).absolute.path,
-          getUserAccessibleAppDataPath: () async {
-            if (!await Permission.storage.request().isGranted) {
-              throw MissingPermissionException(Permission.storage);
-            }
-            return p.join((await getExternalStorageDirectory())!.path);
-          },
-        );
+class AndroidNeonPlatform implements NeonPlatform {
+  const AndroidNeonPlatform();
+
+  @override
+  bool get canUseCamera => true;
```

Comment:
```suggestion
  bool canUseCamera = true;
```

Works as well

Reply:
this then would also need to be made final and our linting doesn't like field initializes in const classes

Reply:
Ahhh